libavformat/whip.c

```diff
@@ -1496,6 +1496,9 @@ static int create_rtp_muxer(AVFormatContext *s)
     char buf[64];
     WHIPContext *whip = s->priv_data;
 
+    if (!(whip->udp->flags & AVIO_FLAG_NONBLOCK)) 
+        whip->udp->flags |= AVIO_FLAG_NONBLOCK;
+
```

Comment:
Either place is fine, what I meant is that the if() itself here is not needed.
Simply `whip->udp->flags |= AVIO_FLAG_NONBLOCK;` will do it, since it's faster than the check if it's unset itself.